docs/home/howto/LOGGING.md

```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
 ```
 After applying this policy every new index created under this one will apply to it. There is also possibility to apply policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
+## How to export Kibana reports to csv format
```

Comment:
```suggestion
## How to export Kibana reports to CSV format
```

Reply:
I would leave 'How to' since it's 'how to' section and we have structure like this here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md